src/main/java/de/craften/ui/swingmaterial/MaterialButton.java

```diff
@@ -154,16 +197,16 @@ protected void paintComponent(Graphics g) {
 
         if (isEnabled()) {
             g2.setColor(getBackground());
-            g2.fill(new RoundRectangle2D.Float(0, 0, getWidth() - offset_lr, getHeight() - offset_td, 3, 3));
+            g2.fill(new RoundRectangle2D.Float(0, 0, getWidth() - offset_lr, getHeight() - offset_td, borderRadius*2, borderRadius*2));
```

Comment:
Because `RoundRectangle2D` receives arc lengths, that is the diameter. Multiply the radius * 2 and you get the diameter.